src/js/js-loader.ts

```diff
@@ -33,9 +33,9 @@ function jsLoader(source: string, sourceMaps: any): void {
 }
 
 function extractTranslations(loader: TranslateLoaderContext, source: string, sourceMaps: any): void {
-    const options = loaderUtils.parseQuery(loader.query);
+    const options = (loaderUtils as any).getOptions(loader) || {};
```

Comment:
Yes, it can be an object or string. [Source](https://github.com/webpack/loader-utils/blob/master/lib/getOptions.js)

How is the best way to handle this in typescript?